lib/email_transactional/version.rb

```diff
@@ -0,0 +1,3 @@
+module EmailTransactional
+  VERSION = '0.1.0'
```

Comment:
Freeze mutable objects assigned to constants.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.